kafka-client/src/main/java/dev/responsive/db/CassandraFactSchema.java

```diff
@@ -308,11 +313,9 @@ public KeyValueIterator<Bytes, byte[]> all(
     throw new UnsupportedOperationException("all is not supported on Idempotent schemas");
   }
 
-  private static ByteBuffer metadataKey(final int partition) {
-    final ByteBuffer key = ByteBuffer.wrap(new byte[METADATA_KEY.get().length + Integer.BYTES]);
-    key.put(METADATA_KEY.get());
-    key.putInt(partition);
-    return key.position(0);
+  // Visible for Testing
+  static String metadataTable(final String tableName) {
+    return tableName + "_md";
```

Comment:
This should never change, right? Could we make this private and then hard-code this expectation in the tests to verify this?

Reply:
yeah we could do that